### PR TITLE
add note about session cookie

### PR DESCRIPTION
**Pull request by henrikac**
This PR adds a small note about session cookie not being set to `secure` and `httpOnly` by default. This was something that I randomly noticed when I was figuring out how to set up authentication and I thought this might be a helpful note to others.

### CODE REVIEW

File: docs/security/authentication.md [hunk 1]
```diff
@@ -748,6 +748,9 @@ These middlewares do the following:
 * the sessions middleware takes the session cookie provided in the request and converts it into a session
 * the session authenticator takes the session and see if there is an authenticated user for that session. If so, the middleware authenticates the request. In the response, the session authenticator sees if the request has an authenticated user and saves them in the session so they're authenticated in the next request.
 
+!!! note
+    The session cookie is not set to `secure` and `httpOnly` by default. Check Vapor's [Session API](../advanced/sessions.md#configuration) how to configure cookies.
```
Comment (jdmcd) [inline docs/security/authentication.md:752]:
```suggestion
    The session cookie is not set to `secure` and `httpOnly` by default. Check Vapor's [Session API](../advanced/sessions.md#configuration) for more information on how to configure cookies.
```

**Review by jdmcd — CHANGES_REQUESTED** [review]
Thank you! One grammatical suggestion

File: docs/security/authentication.md (hunk 1 above)
Comment (henrikac) [reply docs/security/authentication.md]: Thank you for the suggestion :)

**Review by jdmcd — APPROVED** [review]

**Comment by jdmcd** [comment]
@0xTim @gwynne do we merge despite the dead link CI failure? I see other PRs have been merged in that state so I'm not sure.

**Comment by 0xTim** [comment]
@jdmcd I'm planning on fixing the API docs this weekend - if I don't do it by then, then we can merge

**Comment by henrikac** [comment]
Any update on if this can be merged or not? :) @jdmcd @0xTim

**Comment by 0xTim** [comment]
Yeah we can merge this, I've hit a crash with DocC that's blocking the API docs.

Thanks for the PR!

**Pull request merged**